commands/moderation/ban.js

```diff
@@ -0,0 +1,102 @@
+const Discord = require('discord.js');
+var dateFormat = require('dateformat');
+
+module.exports = {
+    name: "ban",
+    description: "Ban a user",
+    
+    execute(msg, con, args) {        
+        const { status, err, toBan, reason } = validBan(msg);
+        if (!status) {
+            return msg.reply(err);
+        }
+
+        banUser(msg, toBan, reason)
+        banSQL(msg, toBan, reason, args, con)
+        banEmbed(msg, toBan, reason)
```

Comment:
I like how this command and kick are broken up so the `execute` function is easy to read.  I'd like to see the same in tempban and unban as well, but we can always do that in a separate issue/PR if you want.

Reply:
Alright, I'll do it here.